src/main/java/team/creative/creativecore/common/util/math/box/BoxFace.java

```diff
@@ -113,7 +129,7 @@ public Boolean isFacingOutwards(boolean first, boolean inverted, Vec3f normal) {
             return outwardOne;
         if (outwardOne != null && outwardTwo == null)
             return outwardOne;
-        if (outwardOne == null && outwardTwo != null)
+        if (outwardOne == null)
```

Comment:
At this point of the checks, outwardTwo is not null
Because if first check was not fulfilled (outwardOne is null and outwardTwo is not null), the second checks what check was not fulfilled (outwardOne is null), making implicit outwardTwo is not null



Basically, the not null check was already done and the second check is just redundant

Reply:
## First Check: `if (outwardOne == outwardTwo)`
At this stage, we not only verify if `outwardOne` equals `outwardTwo`, but also check if `null == null`. If they don't match, it indicates that one of them is not null.

## Second Check: `if (outwardOne != null && outwardTwo == null)`
If they don't match, it implies a difference in value. Failure to meet this condition suggests that either `outwardOne` is null or `outwardTwo` is not null.

## Third Check: `if (outwardOne == null && outwardTwo != null)`
At this juncture, we identify the issue encountered in the second check: `outwardOne` is null. However, as previously mentioned, the comparison `if (outwardOne == outwardTwo)` also encompasses the scenario `null == null`. Therefore, if `outwardOne` is null, `outwardTwo` cannot be null, as the first condition would have been met, rendering the null check for `outwardTwo` redundant.